rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +19,25 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+                    raise serializers.ValidationError(
+                        ValidationErrorMessage(
+                            msg,
+                            code='authorization')
```

Comment:
These lines demonstrate a bit of the weirdness of `ValidationErrorMessage ` to me. This is the same as:

    raise serializers.ValidationError(msg, code='authorization')

Right? Or not?